share/spice/rand_word/rand_word.js

```diff
@@ -31,6 +33,24 @@
                     moreAt: true
                 }
             }
-        });
+        };
+        
+        if ( list_of_list.length == 1 ) {
```

Comment:
This isn't correct. `list_of_list` is an array of 4 element arrays. So right now, if you query `2/3/4 random words` we're only showing a single word now. We need to check the number of total results we received and make sure it's only 1